criu/files-reg.c

```diff
@@ -2008,11 +2013,12 @@ static bool validate_file(const int fd, const struct stat *fd_status, const stru
 
 int open_path(struct file_desc *d, int (*open_cb)(int mntns_root, struct reg_file_info *, void *), void *arg)
 {
-	int tmp, mntns_root, level = 0;
+	int tmp = -1, mntns_root, level = 0;
 	struct reg_file_info *rfi;
 	char *orig_path = NULL;
 	char path[PATH_MAX];
 	int inh_fd = -1;
+	int ret;
```

Comment:
The usage of `tmp` and `ret` is slightly confusing here because we have `return tmp;` and `ret` is used as temporary variable.

Reply:
I believe that a better name for "tmp" is "fd" or "tmp_fd", also I think "ret" is in a proper place here, though it may be a bit confusing.

So I don't want to touch it as:
- "tmp" was named "tmp" for a long time
- patch goal is a bit unrelated - it is to fix error handling so that hidden errors became finally visible, not to fix variable naming
- Using "ret" as a temporary variable name to check called function ret code is a common practice, ret is not always a return value and it's ok (more over we tend to use "exit_code" name for return values in multiple places)

Reply:
I agree with you but what do you think about renaming "tmp" in a separate patch to improve readability?